example/image-classification/symbols/vgg.py

```diff
@@ -62,8 +62,12 @@ def get_symbol(num_classes, num_layers=11, batch_norm=False, dtype='float32', **
                 13: ([2, 2, 2, 2, 2], [64, 128, 256, 512, 512]),
                 16: ([2, 2, 3, 3, 3], [64, 128, 256, 512, 512]),
                 19: ([2, 2, 4, 4, 4], [64, 128, 256, 512, 512])}
-    if not vgg_spec.has_key(num_layers):
-        raise ValueError("Invalide num_layers {}. Possible choices are 11,13,16,19.".format(num_layers))
+    if sys.version_info > (3,):
+        if not num_layers in vgg_spec:
```

Comment:
This works for both py2 and 3. No need to switch

Reply:
Hi,
Thanks for reply.
The has_key method will trigger a run time error at least for Python3.6. I therefore changes the script as this PR. 
Thanks.

Reply:
yes. but `if num_layers not in vgg_spec` works for both py2 and 3